velox/connectors/hive/HiveDataSource.cpp

```diff
@@ -16,6 +16,7 @@
 
 #include "velox/connectors/hive/HiveDataSource.h"
 
+#include <fmt/ranges.h>
```

Comment:
nit for header file ordering: move it under STL header files.

Reply:
@zuyu In Velox, the corresponding .h shall be put at the beginning in its own group. See the dicussion in https://github.com/facebookincubator/velox/issues/1918